packages/lib/src/__tests__/input-builder.test.ts

```diff
@@ -5,20 +5,24 @@ import { buildInputProcessor } from '../input-builder'
 import { IOFormat } from '../io-util'
 import { SmartThingsCommandInterface } from '../smartthings-command'
 import { DefaultTableGenerator } from '../table-generator'
-import { SimpleType } from './io-util.test'
 
+import { SimpleType } from './io-util.test'
 
-jest.mock('../input')
-jest.mock('../smartthings-command')
 
-function buildMockCommand(): SmartThingsCommandInterface {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function buildMockCommand(flags: { [name: string]: any } = {}, profileConfig: { [name: string]: any } = {}): SmartThingsCommandInterface {
```

Comment:
Maybe. I wasn't sure yet though which is why I didn't put it there yet.

The main thing that gave me pause with regards to putting this in `testlib` was that then we would then need to make `lib` (this code) depend on `testlib` which would create a circular dependency between the two. I don't know if that's a bad thing but it definitely makes me uneasy.

I see `testlib` as more of a library for writing tests for commands (i.e. the `cli` package) rather than this code (i.e. the `lib` package), though there will be some overlap.